frontend/src/app/home/home.page.ts

```diff
@@ -1,8 +1,15 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-home',
   templateUrl: 'home.page.html',
   styleUrls: ['home.page.scss'],
 })
-export class HomePage {}
+export class HomePage {
+  constructor(private router: Router) {}
+
+  onStartApplication(): void {
```

Comment:
As in the comment on the html file: Please rename it to "startApplication" or "startJobApplication" or "applyNow" or whatever.

Reply:
We have to discuss about naming conventions for properties, methods. We will first discuss and hear from every one. It is important to discuss ASAP before the code base becomes large. After discussion, If anythings needs to be changed I will change.

Reply:
Same as in the other comment. (Summary: https://angular.io/guide/styleguide#dont-prefix-output-properties )